models/DataObject/ClassDefinition/Data/Multiselect.php

```diff
@@ -122,8 +122,13 @@ public function getRenderType(): ?string
         return $this->renderType;
     }
 
-    public function setDefaultValue(array $defaultValue): static
+    public function setDefaultValue(array|string|null $defaultValue): static
     {
+        if (empty($defaultValue)) {
```

Comment:
The variable always exists
```suggestion
        if (!$defaultValue) {
```

Reply:
Is `'0'` not allowed as default value?

Reply:
mhh true @jdreesen , should be `== null` then?